### ISSUE TITLE
client.login does not return on invalid token

### ISSUE THREAD

**Issue by Benny-** [1Computer1/discord-akairo#8]
Reproducible steps:

```javascript
const client = new AkairoClient({
    ownerID: '123456679',
    prefix: '$',
    commandDirectory: './commands/',
    inhibitorDirectory: './inhibitors/',
    listenerDirectory: './listeners/'
});
 
client.login('foo').then(() => {
    console.log('Started up!');
}, () => {
    console.error('Failed to startup. :(')
});
```

Expected result:
`Failed to startup` is printed and program exits.

Actual result:
`Failed to startup` is never printed. Nothing is printed. Program does not exit.

**Comment by 1Computer1** [comment]
Issue with discord.js's 11.1 release. Fixed in their master/11.1-dev branches.

**Comment by Benny-** [comment]
The "Failed to startup" is called in latest discord.js master. But [another bug](https://github.com/hydrabolt/discord.js/issues/1685) prevents it from exiting.
